### PR TITLE
Create elastic search index after service is up

### PR DESCRIPTION
**Pull request by sadmananik**
Farajaland PR: https://github.com/opencrvs/opencrvs-farajaland/pull/584

### CODE REVIEW

File: dev.sh [hunk 1]
```diff
@@ -118,4 +118,4 @@ echo
 sleep 10
 
 yarn dev:secrets:gen
-concurrently "yarn run start" "yarn run compose:deps"
\ No newline at end of file
+concurrently "yarn run start" "yarn run compose:deps" "bash setup-elastic-index.sh"
```
Comment (Zangetsu101) [inline dev.sh:121]: We should add this for the case when we are running only the dependencies with `--only-dependencies`

File: setup-elastic-index.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+# graphic logo are (registered/a) trademark(s) of Plan International.
+
+function wait_for_elasticsearch {
+  local -a args=( '-s' '-D-' '-m15' '-w' '%{http_code}' "http://localhost:9200/" )
```
Comment (euanmillar) [inline setup-elastic-index.sh:12]: This should accept hostname as a var and we need it to be additionally called by the deploy.sh script from Farajaland

**Review by euanmillar — CHANGES_REQUESTED** [review]

File: dev.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (sadmananik) [reply dev.sh:121]: For the deployable solution, I have changed Farajaland restore-metadata.sh and setup-settings.sh file that will create ocrvs index on deployment

Comment (sadmananik) [reply dev.sh:121]: yes. I almost forgot. Thanks.

File: setup-elastic-index.sh (hunk 2 above)
Comment (sadmananik) [reply setup-elastic-index.sh:12]: I have handled it another way. You can check this [PR](You can check https://github.com/opencrvs/opencrvs-farajaland/pull/584)

**Review by rikukissa — APPROVED** [review]

**Pull request merged**